Makefile

```diff
@@ -39,6 +39,19 @@ dev-int:
 ci-int:
 	docker compose -p int -f ./docker-compose.ci.yml run --rm api ci:int
 
+.PHONY: aws-ecs
+aws-ecs:
+	ecs-cli configure profile --profile-name ${ECS_PROFILE_NAME} --access-key ${AWS_ACCESS_KEY_ID} --secret-key ${AWS_SECRET_ACCESS_KEY}
+	ecs-cli configure --cluster ${ECS_CLUSTER_NAME} --default-launch-type ${ECS_LAUNCH_TYPE} --region ${AWS_REGION} --config-name ${ECS_PROFILE_NAME}
+
+.PHONY: aws-compose
+aws-compose:
+	HYPERDX_APP_URL=${HYPERDX_APP_ALB_URL} HYPERDX_APP_PORT=${HYPERDX_APP_ALB_PORT} ecs-cli compose --project-name ${ECS_CLUSTER_NAME} --file ./docker-compose.aws.yml --region ${AWS_REGION} --ecs-params ./aws/ecs-params.yml create --launch-type ${ECS_LAUNCH_TYPE} --create-log-groups
+
+.PHONY: aws-create-service
+aws-create-service:
+	aws ecs create-service --service-name ${ECS_SERVICE_NAME} --cluster ${ECS_CLUSTER_NAME} --cli-input-json file://aws/service-definition.json
+
```

Comment:
Can we move this to `aws/Makefile` instead ?

Reply:
move [aws/Makefile](https://github.com/hyperdxio/hyperdx/pull/79/files#diff-4fea5781d6e25490b90b9a714eb383695183edfbdc3ced0f666e791de27f1b70)